lib/util.test.js

```diff
@@ -0,0 +1,58 @@
+jest.mock("sequelize");
+jest.mock("../sequelize/db");
+jest.mock("../database/postgres/pg");
+jest.mock("../database/elasticsearch/elastic");
+jest.mock("./log");
+
+const sequelize = require("sequelize");
+const db = require("../sequelize/db");
+const pg = require("../database/postgres/pg");
+const es = require("../database/elasticsearch/elastic");
+
+sequelize.Op = { and: "and", lt: "lt" };
+const Accounts = {
+  findAll: jest.fn(),
+};
+db.getModels = () => {
+  return { Accounts: Accounts };
+};
+pg.deletePgAccount = jest.fn();
+es.deleteAccount = jest.fn();
+const logGen = require("./log");
+const logger = {
+  info: jest.fn(),
+  error: jest.fn(),
+};
+logGen.mockReturnValue(logger);
+
+const util = require("./util");
+
+describe("Testing cleanAnonymous function", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it("should call findAll with correct queries", async () => {
+    Accounts.findAll.mockReturnValue([]);
+    await util.cleanAnonymous();
+    const args = Accounts.findAll.mock.calls[0][0];
+    expect(args.attributes[0]).toEqual("id");
+    expect(args.attributes[1]).toEqual("username");
+    expect(Number(args.where.and[0].createdAt.lt)).toBeLessThan(
+      Number(new Date(new Date() - 5 * 24 * 60 * 60 * 1000 + 1))
+    );
+    expect(args.where.and[1].email).toBeNull();
```

Comment:
I wrote some strict test here because I thought that it's important to protect this query. Someone can accidently clear up the whole db by making a little change.

Reply:
NICE. this should be in the comment. 

```
// Hi future engineers, I wrote some strict test here because I thought that it's important to protect this query. Someone can accidently clear up the whole db by making a little change. If you are reading this, please pass down this knowledge
```